plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/TestPhoenixTypeMapping.java

```diff
@@ -133,6 +133,186 @@ public void testBasicTypes()
                 .execute(getQueryRunner(), trinoCreateAsSelect("test_basic_types"));
     }
 
+    @Test
+    public void testBoolean()
+    {
+        SqlDataTypeTest.create()
+                .addRoundTrip("boolean", "true", BOOLEAN, "true")
+                .addRoundTrip("boolean", "false", BOOLEAN, "false")
+                .addRoundTrip("boolean", "NULL", BOOLEAN, "CAST(NULL AS BOOLEAN)")
+                .execute(getQueryRunner(), trinoCreateAsSelect("test_boolean"))
+
+                .addRoundTrip("integer primary key", "1", INTEGER, "1")
```

Comment:
Can we subclass SqlDataTypeTest here to avoid repeating this everywhere?

Reply:
BTW the chaining of `execute().addRoundTrip().execute()` feels easy to overlook.

I'd prefer something like:

```
testCases = SqlDataTypeTest.create().addRoundTrip()...
testCases.execute(getQueryRunner(), trinoCreateAsSelect("test_boolean"));
testCases.addRoundTrip("integer primary key", "1", INTEGER, "1")
         .execute(getQueryRunner(), phoenixCreateAndInsert("tpch.test_boolean"));
```

Reply:
Hm, I don't feel so. It's just one line below and the possibility to overlook is very low in my opinion.